src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -33,6 +33,7 @@ export class AdminSearchPageComponent {
   searchQuery: string = '';
   instructors: InstructorAccountSearchResult[] = [];
   students: StudentAccountSearchResult[] = [];
+  maxResultsToShow: number = 50;
```

Comment:
Should copy this value from back-end

Reply:
Hello! Just to make a quick clarification. I initially tried to retrieve the value from back-end which resides within the `SearchManager` file, noting that it is used to limit the number of results returned from querying the database. 

However, I came to realise that the current design does not pass the value onto the frontend, which merely receives the query results of the database. Yet, in order to show the message prompt, the frontend needs to know when the maximum number of display results has been shown.

So I think the point I wish to clarify is, is it better to keep the 2 numbers separate (I may wish to retrieve 50 results from the database query, but only show 30 on the frontend) or to share the numbers? Admittedly the former is easier because when I first worked on the issue I spent a good amount of time (possibly with the wrong approach but I can try again) trying to pull the value from the back-end. Thank you!

Reply:
It is easy to see why your method here is wrong @tjtanjin. There could exactly be 50 results in the database and the frontend will have no way to tell if there is more results available. The backend should be the one informing the frontend whether there are more results since it is doing the capping. Furthermore, there is duplication - 2 places having the same value instead of one. Ideally a paginated endpoint should return a flag indicating if there is more results. However, this is not straightforward. A couple of interactions have to be modified to get this result propagated back to the frontend.

As you pointed out above, maybe another approach is to have a BE limit and a FE limit. So, backend will return up to 50 results and FE will only show up to 40. This way we know for sure whether there are too many results or not. It is not ideal since now we have 2 limits in the codebase but I think it works for now unless @wkurniawan07 has objections